CIFAR10/inference/laurentmazare_resnet9_1p100_pytorch.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "v1.0",
+    "author": "Laurent Mazare",
+    "authorEmail": "lmazare@gmail.com",
+    "framework": "PyTorch v1.0.0.dev20181116",
+    "codeURL": "https://github.com/LaurentMazare/deep-models/tree/master/cifar-10",
```

Comment:
Can you add the commit hash either to the `codeURL` or as a separate `commitHash` field?

Reply:
Sure, I just pushed a change adding the commit hash to codeURL.